internal/jujuapi/access_control_test.go

```diff
@@ -1427,6 +1427,18 @@ func (s *accessControlSuite) TestCheckRelationControllerAdministratorFlow(c *gc.
 		c.Assert(err, gc.IsNil)
 		c.Assert(res.Allowed, gc.Equals, tc.want)
 	}
+	// Check that the same tuples can be checked via the CheckRelations API
```

Comment:
I'm not the biggest fan of putting this test inside another test. It's a whole separate API method, I think it should get its own test unless you feel strongly that it's fine like this.